### PR TITLE
Restore CI

### PR DESCRIPTION
**Pull request by israel-hdez**
Since TravisCI is no longer enabled, use GitHub actions for CI.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Snyk scan
```
Comment (jmazzitelli) [inline .github/workflows/ci.yml:1]: Is this related to "Snyk"?

Comment (israel-hdez) [reply .github/workflows/ci.yml:1]: Ops! Nope.
Let me change it.

Also, I think there is a syntax error, because I see nothing is running.

**Comment by israel-hdez** [comment]
I'm not sure why it's not running here.
In my fork it runs: https://github.com/israel-hdez/helm-charts/pull/3

**Comment by israel-hdez** [comment]
perhaps since it's new, this needs to be merged?

**Comment by israel-hdez** [comment]
I think this needs to be merged before it can work. I do see a blank check in the "Checks" tab:

image

**Review by jmazzitelli — APPROVED** [review]
